packages/python/plotly/plotly/express/_core.py

```diff
@@ -1576,6 +1581,9 @@ def aggfunc_continuous(x):
     if args["color"]:
         if not args["hover_data"]:
             args["hover_data"] = [args["color"]]
+        elif isinstance(args["hover_data"], dict):
```

Comment:
could we get a test to lock down whatever bug this is correcting?

Reply:
test added